redditfunction.py

```diff
@@ -5,7 +5,8 @@
 class RedditFunction :
     
     #This function returns a list of reddit post links
-    def getPushshiftData(self, size, after, before, queryr) :
+    def getPushshiftData(self, size, after, before, q) :

```

Comment:
Try to look for libraries that make things easier - in this case, PRAW already exists as a Reddit wrapper https://praw.readthedocs.io/en/latest/ - if you can leverage modules that already exist, it saves you debugging time and makes your code more simple

Reply:
I'm getting an error when I try to import praw: 
`ModuleNotFoundError: No module named 'praw'`
I've tried reinstalling, `pip3 install praw --upgrade --force-reinstall`, and I can see the package is there in the site-packages directory in file explorer, but I can't import it in python.